plugins/reporters/freemarker/src/main/kotlin/FreemarkerTemplateProcessor.kt

```diff
@@ -343,8 +343,7 @@ class FreemarkerTemplateProcessor(
          * Return a flag if the given [sourceLocation] refers to the full source file.
          */
         @Suppress("UNUSED") // This function is used in the templates.
-        fun isFullFileLocation(sourceLocation: TextLocation) =
-            sourceLocation.startLine == TextLocation.UNKNOWN_LINE && sourceLocation.endLine == TextLocation.UNKNOWN_LINE
+        fun isFullFileLocation(sourceLocation: TextLocation) = sourceLocation.isUnknown
```

Comment:
@nnobelis @fviernau do you think we could remove / inline this already now? (Also see the commit message.)

Reply:
I'm ok with inlining this, I just proposed an alternative name.

Reply:
I've done this in a new commit to be more easily revertible, just in case.